examples/howto_examples/plot_acoustic_radiations.py

```diff
@@ -49,12 +49,12 @@
 ar_rois = afd.read_ar_templates()
 
 bundles = abd.BundleDict({
-    "AR_L": {
+    "Left Acoustic": {
```

Comment:
Do you want to do this to be consistent with "Left Optic Radiation"
```suggestion
    "Left Acoustic Radiation": {
```